index.bs

```diff
@@ -179,10 +179,10 @@ interface ImageCapture {
 
 <pre class="idl">
   interface PhotoCapabilities {
-    readonly attribute FillLightMode      fillLightMode;
-    readonly attribute MediaSettingsRange imageHeight;
-    readonly attribute MediaSettingsRange imageWidth;
-    readonly attribute sequence&lt;boolean>  redEyeReduction;
+    readonly attribute sequence&lt;boolean>       redEyeReduction;
+    readonly attribute MediaSettingsRange      imageHeight;
+    readonly attribute MediaSettingsRange      imageWidth;
+    readonly attribute sequence&lt;FillLightMode> fillLightMode;
```

Comment:
Actually, a sequence [cannot be an attribute](https://heycam.github.io/webidl/#idl-attributes). It's invalid WebIDL. We'd need:
```js
readonly attribute FrozenArray<boolean>    redEyeReduction;
readonly attribute FrozenArray<boolean>    fillLightMode;
```